content/blog/2018_conference_attendees_useful_info.md

```diff
@@ -0,0 +1,43 @@
+Title: Useful info PyCon India Conference Attendees
+Date: 2018-10-04 21:30
+Category: 2018
+Slug: useful-info-conference-attendees
+
+
+If you are coming to PyCon India 2018 and attending conference, then please read this blog to understand some dos and don'ts along with lot of useful info.
+<!-- PELICAN_END_SUMMARY -->
+
+Hope you're all set for PyCon India 2018.
+
+Here are a few things we would like to bring to your attention.
+
+## Reaching to Hyderabad:
+Please find information at https://in.pycon.org/2018/travel.html.
+
+## Reaching the venue (HICC):
+For attendees using public transport, shuttles will be available between Hitex Charminar (Arch)  and the venue - [HICC](https://goo.gl/maps/SrZKN3PhAau). For any further guidance, please reach out to our transport team @ Subhash - 9963754460
+
+## At the Registration Desk:
+The first step is to register and collect your kit from the front desk. Please carry your ticket as the team would be looking for the registration number on it.  The queues will be organized by Registration Number.
+![ticket image](https://github.com/pythonindia/inpycon2018/blob/master/img/ticket-image.jpeg)
+
+## Lightning Talks:
+* On each of the conference days, we are conducting lightning talks and we encourage you to participate.For those who would like to talk, please ask for the form at the registration desk and drop it in the drop box near the registration desk. Organizers might have to curate, in case of an overwhelming response. All selected speakers will be reached out to by 2:30 PM on both days.
+* Of course, you are always welcome to reach out to the audience using open spaces, for further details look at [@openspaces](https://in.pycon.org/2018/open-spaces.html).
+
+## During the conference:
+* Want to meet new people and learn to explore python in wholly different ways? Then don’t forget to check out the open spaces on Hall 3 and Hall 4 on the second floor. Please look [@openspaces](https://in.pycon.org/2018/open-spaces.html) for further details
```

Comment:
Can we remove Hall 3 and hall 4 and just mention second floor here

Reply:
Done.